test/retry.js

```diff
@@ -124,7 +131,7 @@ test('respect 413 Retry-After with timestamp', async t => {
 	});
 
 	const result = await ky(server.url).text();
-	t.true(Number(result) >= retryAfterOn413 * 1000);
+	t.true(Number(result) >= (retryAfterOn413 - 1) * 1000);
```

Comment:
Yes, we need `-1`. To round the result. You can be +500ms into the next second and the result can be `1500`ms, which is still valid.